node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,22 @@
+const filename = process.argv[2];
+
+const fs = require('fs');
```

Comment:
Although this will work, it is best to always have all `require` statements as the first things in the file